app/controllers/landing_page_controller.rb

```diff
@@ -123,8 +123,14 @@ def preview
     begin
       structure = CLP::LandingPageStore.load_structure(cid, preview_version)
 
-      # Uncomment to use static data instead of dynamic from DB
-      # structure = JSON.parse(CustomLandingPage::ExampleData::DATA_STR)
+      # if static landing page enabled, load data from config/initializers/landing_page.rb
+      if (APP_CONFIG.clp_static_enabled)
```

Comment:
this whole block seems unnecessary, structure is set to the correct value already in `app/services/custom_landing_page/landing_page_store_static.rb` when you call `load_structure`

IMHO you can just update the comment section to say that one should run the rake task and set clp_static_enabled: true